static/js/StaticPages.jsx

```diff
@@ -3126,6 +3129,330 @@ const JobsPage = memo(() => {
+        loadProducts();
+    }, []);
+
+    console.log("products: ", products)
+
+    // Generalized function for catching products page analytics
+    const productsAnalytics = (rank, product_cta, label, event) => {
+        gtag("event", `${product_cta}_${event}`, {
+          panel_type: "strapi-static",
+          panel_number: 0,   
+          panel_name: "Products Page",
+          position: rank,
+          experiment: 1,
+          feature_name: label,
+          engagement_type: "static",
+          engagement_value: 0,
+        });
+    }
+
+
+    // The static content on the page inviting users to browse our "powered-by" products
+    const DevBox = () => {
+        return (
+          <div className='productsDevBox'>
+            <p className='productsDevHeader'>
+                <span className="int-en">Powered by Sefaria</span>
+                <span className="int-he">פרויקטים מכח ספריא</span>
+            </p>
+            <p>
+                <span className="int-en">
+                Check out the products our software developer friends from around the world have been building for you! <a href="www.example.com">Explore</a>
+                </span>
+                <span className="int-he">נסו את המוצרים שמפתחי תוכנה וידידי ספריא מרחבי העולם בנו עבורכם! <a href="www.example.com">גלו את הפרויקטים</a></span>
+                
+            </p>
+          </div>
+        );
+      };
+
+    /**
+     * The following are the building block components of an individual product. 
+     */
+     
+    // The title and gray background label for each product
+    const ProductTitle = ({product}) => {
+        return (
+            <div className='productsTitleAndLabel'>
+            <span className="productsTitle">
+                <span className="int-en">{product.titles.en}</span>
+                <span className="int-he">{product.titles.he}</span>
+            </span>
+            {product.type.en ? (<span className="productsTypeLabel">                                
+                <span className="int-en">{product.type.en}</span>
+                <span className="int-he">{product.type.he}</span>
+            </span>) : ''}
+        </div>
+        );
+    };
+
+
+
+    // The call-to-action (link) in the heading of each product
+    const ProductCTA = ({product}) => {
+        return (
+            <div className="cta">
+                    {product.ctaLabels?.map(cta => (
+                    <OnInView onVisible={() => productsAnalytics(product?.rank, `${product?.titles.en}_${cta.text.en}`, product?.type.en, "viewed")}>
+
+                        <a onClick={productsAnalytics(product?.rank, `${product?.titles.en}_${cta.text.en}`, product?.type.en, "clicked")} 
+                            href={cta.url} 
+                            key={cta.text.en}>
+                            {cta.icon.url && <img className="productsCTAIcon" 
+                                                data-image-path={cta.icon.url} 
```

Comment:
What is this for?

Reply:
Sorry, I'm not clear. What specifically?

Reply:
Removed :)